docs/PAI-ADAPTATIONS.md

```
@@ -192,6 +194,161 @@ export function fileLog(message: string, level = "info") {
+
+The plugin now loads only the pieces OpenCode's native system cannot provide:
+- `USER/ABOUTME.md` — personal background
+- `USER/TELOS/TELOS.md` — goals and mission
+- `USER/DAIDENTITY.md` — DA name and personality
+- `USER/AISTEERINGRULES.md` — behavioral steering rules
+
+PAI Core (Algorithm, ISC, Capabilities) loads automatically via the skill system.
+
+### v3.0 Architecture — Context Flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)